api.raml

```diff
@@ -193,6 +186,30 @@ schemas:
             "required": false,
             "type": "string",
             "enum": ["yes", "no"]
+          },
+          "password": {
```

Comment:
yuk!  https://github.com/silinternational/idp-id-broker/commit/2b73555c3f2ae1d709b86a7d14fd8553761c482a 

We never put any conditional logic in place to suppress the password info on an update...is there some reason we'd want something like that?